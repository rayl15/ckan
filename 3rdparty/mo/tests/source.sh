#!/usr/bin/env bash

cd "${0%/*}"
cat <<EOF | ../mo --source=source.vars
{{VAR}}
{{#ARR}}
* {{.}}
{{/ARR}}
{{ASSOC_ARR.a}} {{ASSOC_ARR.b}}
EOF
