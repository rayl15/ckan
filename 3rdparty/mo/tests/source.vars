export VAR=value
export ARR=(1 2 3)
declare -A ASSOC_ARR
export ASSOC_ARR=([a]=AAA [b]=BBB)