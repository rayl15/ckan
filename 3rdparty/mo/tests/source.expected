value
* 1
* 2
* 3
AAA BBB
